src/blob/core.rs

```diff
@@ -262,28 +262,22 @@ where
         Ok(header)
     }
 
-    pub(crate) async fn read_last(
+    #[allow(dead_code)]
```

Comment:
Why this allow is needed?

Reply:
`read_latest` is no longer used. Since this request `Storage` uses `get_latest_entry`. I thought it might be worth keeping this function for the future. What do you think?

Reply:
I think it's better to just keep in git history since it can make it harder to refactor code

Reply:
I've removed this function

Reply:
What about other dead_code allowances, they should be there?

Reply:
If we assume that the only way to read data now is by getting `Entry` first, then we should keep `read_all_entries`. 
`contains` is questionable, but as long as it is implemented as a simple redirection to `get_latest_entry`, we can keep it. It is good to have such function just as a declaration, that the struct supports fast checking for existence.

Reply:
I removed all dead code